libraries/botbuilder-dialogs-adaptive-testing/src/testScript.ts

```diff
@@ -98,43 +111,71 @@ export class TestScript extends Configurable implements TestScriptConfiguration
         }
     }
 
+    /**
+     * Build default test adapter.
+     * @param testName Name of test.
+     * @param middlewares Middlewares to be added to the adapter.
+     * @returns Test adapter.
+     */
+    public defaultTestAdapter(testName?: string, ...middlewares: Middleware[]): TestAdapter {
+        const storage = new MemoryStorage();
+        const userState = new UserState(storage);
+        const convoState = new ConversationState(storage);
+
+        const adapter = new TestAdapter(TestAdapter.createConversation(testName));
+        useBotState(adapter, userState, convoState);
+        middlewares.forEach((middleware) => {
+            adapter.use(middleware);
+        });
+
+        adapter.onTurnError = async (context, err) => {
```

Comment:
This is a nit of mine, but this could more succinctly be written as `adapter.onTurnError = (context, err) => context.sendActivity(err.message)`. No need to explicitly mark as async or await the result.

Reply:
Thx, I tried this but the return type of `onTurnError` is `Promise<void>` and the return type of `context.sendActivity(err.message)` is `Promise<ResourceResponse>`, how can I fix the mismatch issue?